assets/src/application/ui/input/InlineEditInput/InlineEditText.tsx

```diff
@@ -33,14 +33,14 @@ const Preview: React.FC<InlineEditPreviewProps> = ({ fitText, isEditing, onReque
 	return textInput;
 };
 
-const InlineEditText: React.FC<TextProps> = ({ fitText = true, placeholder = '', tag: as, ...props }) => {
+const InlineEditText: React.FC<TextProps> = ({ fitText = true, placeholder = '', tag: as, tooltip, ...props }) => {
 	return (
 		<InlineEdit
 			{...props}
 			as={as}
 			inputType='text'
 			placeholder={placeholder}
-			Preview={(previewProps) => <Preview {...previewProps} fitText={fitText} />}
+			Preview={(previewProps) => <Preview {...previewProps} fitText={fitText} tooltip={tooltip} />}
```

Comment:
is this prop name on `InlineEdit` supposed to be Capitalized?

should it not be:

```suggestion
			preview={(previewProps) => <Preview {...previewProps} fitText={fitText} tooltip={tooltip} />}
```

or is it because it gets rendered inside `InlineEdit` as if it was an imported component?

[EDIT] ya I jsut saw that the other inputs are doing the same thing, so that must be the case

Reply:
that is from the type:
```

export interface InlineEditProps extends Partial<ChakraEditableProps>, CommonInputProps<HTMLInputElement> {
	inputType?: InputType;
	Preview?: React.ComponentType<InlineEditPreviewProps>;
}
```